README.md

```diff
@@ -171,7 +156,7 @@ Example:
 
 ```puppet
 class { 'letsencrypt::plugin::dns_rfc2136':
-  server     => '1.2.3.4',
+  server     => '192.0.2.1',
```

Comment:
This is TEST-NET-1 (192.0.2.0/24) from [rfc5737](https://datatracker.ietf.org/doc/rfc5737/).